packages/terra-popup/src/_PopupOverlay.jsx

```diff
@@ -59,6 +59,8 @@ class PopupOverlay extends React.Component {
           onClick={this.handleOnClick}
           {...customProps}
           className={cx(['popup-overlay', customProps.className])}
+          // eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
+          tabIndex="0"
```

Comment:
Doesn't adding tab index here allow users to tab to the overlay?

Reply:
Removed in 4bc7213